.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,4 +30,3 @@ Please remove all except for everything applicable, and then remove this line.
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] My changes generate no new warnings
 - [ ] Docs have been updated if applicable
-- [ ] Tests have been updated if applicable
```

Comment:
These checklists create an `X of Y tasks` preview, like so:
![image](https://user-images.githubusercontent.com/1634972/224686961-14bee053-b9f4-4971-bab7-0840b27635e3.png)

This is a bit misleading. Thoughts on doing away with the checkboxes or rephrasing the items so that they should all be checked before submitting? eg.

- [ ] Changes have new/updated automated tests, if applicable
- [ ] Changes have new/updated docs, if applicable
​
- [ ] I have performed a self-review of my own code
- [ ] I have added comments on any new, hard-to-understand, code
- [ ] Automated tests pass
- [ ] Changes have been manually tested
- [ ] Changes generate no new errors or warnings

Reply:
sure! down for this